cfgov/tccp/jinja2/tccp/card.html

```diff
@@ -254,16 +265,16 @@ <h2 class="h3 u-mb15">
             <p>
                 The purchase APR is
                 {% if 'V' in card.index %}
-                    variable, meaning it can go up or down based on an index. You can
-                    <a href="/ask-cfpb/what-is-the-difference-between-a-fixed-apr-and-a-variable-apr-en-45/">learn
-                    more about variable APRs in general</a> or expand this section to
-                    learn more about the index this APR is based on.
+                    variable, meaning it can go up or down based on the index
+                    listed in the details below.
```

Comment:
I wonder if users will understand what we mean by "in the details below". The previous text that suggested expanding this section seemed a little more useable to me. I don't have a great suggestion but wanted to mention in case you have any ideas for how to keep that directive.

Reply:
Let's keep an eye on this in user testing to see how (or if) people are interacting with those expandables. We can reword depending on what we see.